app/src/main/java/com/io/moviO/movieDetails/MovieDetailsFragment.kt

```diff
@@ -21,17 +23,18 @@ class MovieDetailsFragment : Fragment(R.layout.fragment_movie_details) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         binding = FragmentMovieDetailsBinding.bind(view)
-        requireArguments().getString(ARG_MOVIE_ID)?.let { viewModel.getMovieById(it) }
+        requireArguments().getInt(ARG_MOVIE_ID).let { viewModel.getMovieById(it) }
         viewModel.movie.observe(viewLifecycleOwner, Observer {
             when (it) {
                 is DataResult.Success ->
                     binding.apply {
-                        movieName.text = it.value.name
-                        movieYear.text = it.value.year
-                        movieGenre.text = it.value.gerne
-                        movieCast.text = it.value.cast
+                        movieName.text = it.value.title
+                        movieReleaseDate.text = it.value.releaseDate
+                        movieGenre.text = convertGenres(it.value.genres!!)
+                        movieRating.text = it.value.voteAverage.toString().plus("/10")
```

Comment:
I think if you do it like this `"${it.value.voteAverage}/10"` it will be more readable